_tuning-your-cluster/availability-and-recovery/snapshots/sm-dashboards.md

```diff
@@ -1,11 +1,15 @@
 ---
 layout: default
-title: Snapshot management
-parent: Index and snapshot management in OpenSearch Dashboards
-nav_order: 17
+title: Snapshot management in Dashboards
```

Comment:
I vote for moving this back to Dashboards and linking from here.

Reply:
I do too. Imma change it